src/vaadin-combo-box-mixin.html

```diff
@@ -262,6 +262,17 @@
       this._observer = new Polymer.FlattenedNodesObserver(this, info => {
         this._setTemplateFromNodes(info.addedNodes);
       });
+
+      const bringToFrontListener = (e) => {
+        if (this.opened && this.$.overlay.$.dropdown.$.overlay.bringToFront) {
```

Comment:
Why `this.opened` is checked here (the same change to date-picker didn't have it)?

Reply:
I just make sure the dropdown of combo-box opened already then bringToFront is invoked. but I checked and it is not good, because if we open dropdown of cb-box like this `fireMousedownMouseupClick(combobox.inputElement.root.querySelector('label'));`, 
The overlay of combo-box is not opened, it needs some times to open, and bringToFront is not invoked, and `this.$.overlay.$.dropdown.$.overlay.bringToFront` has error.

So I refactor the code like this: `const bringToFrontListener = (e) => {
        requestAnimationFrame(() => {
          if (this.$.overlay.$.dropdown.$.overlay.bringToFront) {
            this.$.overlay.$.dropdown.$.overlay.bringToFront();
          }
        });
      };`
Please corrrect me if I am wrong.

Reply:
Now the change is closer to that of `vaadin-date-picker`'s, but for some reason the if-check for `bringToFront` is inside the requestAnimationFrame in this one.

Reply:
Ah, gotcha. The overlay is lazily instantiated on open so you get the error.